common/src/main/java/com/wynntils/mc/utils/RenderUtils.java

```diff
@@ -31,15 +32,15 @@ public class RenderUtils {
 
     public static final ResourceLocation highlight = new ResourceLocation("wynntils", "textures/highlight.png");
 
-    public static void drawRect(int color, int x, int y, int width, int height) {
+    public static void drawRect(CustomColor color, int x, int y, int width, int height) {
         drawRect(new PoseStack(), color, x, y, 0, width, height);
     }
 
-    public static void drawRect(PoseStack poseStack, int color, int x, int y, int z, int width, int height) {
-        int alpha = (color >> 24) & 0xFF;
-        int red = (color >> 16) & 0xFF;
-        int green = (color >> 8) & 0xFF;
-        int blue = color & 0xFF;
+    public static void drawRect(PoseStack poseStack, CustomColor color, int x, int y, int z, int width, int height) {
+        int alpha = color.a & 0xFF;
```

Comment:
Can't we guarantee that a custom color value is always below 255? Or is it just a bit of extra defensive programming?